pkg/registry/service/registry_test.go

```diff
@@ -316,89 +319,6 @@ func TestEtcdWatchServices(t *testing.T) {
 	watching.Stop()
 }
 
-func TestEtcdWatchServicesBadSelector(t *testing.T) {
```

Comment:
As explained above - see separation into 2 commits for explanation for what is happening here (he first commit is what explains it).